src/DIRAC/Core/Utilities/ObjectLoader.py

```diff
@@ -177,6 +177,11 @@ def getObjects(self, modulePath, reFilter=None, parentClass=None, recurse=False,
 
 def loadObjects(path, reFilter=None, parentClass=None):
```

Comment:
Add the deprecated decoarator? Or do we still need it?

Reply:
It's still being used in a few place. I could probably find a proper fix, but that's for a major release